src/Tools/dotnet-dump/Commands/SOSCommandForWindows.cs

```diff
@@ -0,0 +1,24 @@
+using Microsoft.Diagnostic.Repl;
+using System.CommandLine;
+
+namespace Microsoft.Diagnostic.Tools.Dump
+{
+    [Command(Name = "dumprcw",              AliasExpansion = "DumpRCW",             Help = "Displays information about a Runtime Callable Wrapper.")]
+    [Command(Name = "dumpccw",              AliasExpansion = "DumpCCW",             Help = "Displays information about a COM Callable Wrapper.")]
+    [Command(Name = "dumppermissionset",    AliasExpansion = "DumpPermissionSet",   Help = "Displays a PermissionSet object (debug build only).")]
+    [Command(Name = "traverseheap",         AliasExpansion = "TraverseHeap",        Help = "Writes out a file in a format understood by the CLR Profiler.")]
```

Comment:
Is this command new? I don't recall it.
What profiler is "The CLR profiler"?

Reply:
Here is all I known about it (from our SOS docs). I tried to summarize this description: 

|Command|Description|
|-------------|-----------------|
|**TraverseHeap** [**-xml**] \<*filename*>|Writes heap information to the specified file in a format understood by the CLR profiler. The **-xml** option causes the **TraverseHeap** command to format the file as XML.<br /><br /> You can download the CLR Profiler from the [Microsoft Download Center](https://go.microsoft.com/fwlink/?LinkID=67325).|

Reply:
So its not new? I guess I managed to overlook it all these years. Thanks Mike!